server/protocols/jmap/src/main/java/org/apache/james/jmap/JMAPConfiguration.java

```diff
@@ -0,0 +1,106 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.jmap;
+
+import java.util.Optional;
+
+import org.apache.james.util.Port;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+
+public class JMAPConfiguration {
+
+    // TODO move tests too
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private Optional<Boolean> enabled = Optional.empty();
+        private Optional<Boolean> wiretap = Optional.empty();
```

Comment:
What does it mean? is that specific to Netty? If yes, how can we use it in a jetty module(jmap-draft) ?

Reply:
wiretap replaces the logbook

Reply:
we took jetty out of jmap-draft, don't get your question

Reply:
https://projectreactor.io/docs/netty/release/reference/index.html#_wire_logger

Reply:
I meant this JMapConfiguration is served common properties of a JMAP server like port and host, enabled...  while wiretap is something specific to Netty.

Reply:
BTW this comment is not critical

Reply:
It is specific to Netty yes, but I still don't get your concern as the goal is to take Jetty out for JMAP?